crypto/bio/bss_file.c

```diff
@@ -7,9 +7,6 @@
  * https://www.openssl.org/source/license.html
  */
 
-#ifndef HEADER_BSS_FILE_C
-# define HEADER_BSS_FILE_C
-
 # if defined(__linux) || defined(__sun) || defined(__hpux)
```

Comment:
You should adjust the indent of all preprocessor lines that were guarded.

Reply:
(not only here, I just noticed...)

Reply:
Done (master and 1.1.1)